prov/efa/src/efa.h

```diff
@@ -340,25 +340,10 @@ ssize_t efa_cq_readfrom(struct fid_cq *cq_fid, void *buf, size_t count, fi_addr_
 ssize_t efa_cq_readerr(struct fid_cq *cq_fid, struct fi_cq_err_entry *entry, uint64_t flags);
 
 static inline
-bool efa_ep_support_rdma_read(struct fid_ep *ep_fid)
-{
-	struct efa_ep *efa_ep;
-
-	if (!rxr_env.use_device_rdma)
-		return 0;
-
-	efa_ep = container_of(ep_fid, struct efa_ep, util_ep.ep_fid);
-	return efa_ep->domain->device->device_caps & EFADV_DEVICE_ATTR_CAPS_RDMA_READ;
-}
-
-static inline
-bool efa_ep_support_rnr_retry_modify(struct fid_ep *ep_fid)
+bool efa_ep_support_rnr_retry_modify(struct efa_domain *domain)
```

Comment:
this function name should be `efa_domain_support_rnr_retry_modify()` and should be moved to `efa_domain.h`

Reply:
done